etcdserver/server.go

```diff
@@ -153,10 +153,14 @@ type Server interface {
 
 // EtcdServer is the production implementation of the Server interface
 type EtcdServer struct {
-	// r must be the first element to keep 64-bit alignment for atomic
+	// r and inflightSnapshots must be the first elements to keep 64-bit alignment for atomic
 	// access to fields
 	r raftNode
 
+	// count the number of inflight snapshots.
+	// MUST use atomic operation to access this field.
+	inflightSnapshots int64
```

Comment:
this should be above raftNode for 8-byte alignment; sizeof(raftNode) == 100 on 386